src/expressions/StaticContext.ts

```diff
@@ -37,6 +37,7 @@ export type FunctionDefinition = {
  */
 export default class StaticContext {
 	public parentContext: IContext;
+	private _registeredDefaultFunctionNamespace: string = 'http://www.w3.org/2005/xpath-functions';
```

Comment:
It doesn't really matter but we could just make this one public instead of adding a separate getter and setter.
Your choice.